widgetastic_manageiq.py

```diff
@@ -425,3 +431,145 @@ class Accordion(PFAccordion):
     def is_dimmed(self):
         return bool(
             self.browser.elements('.//div[contains(@id, "tree") and contains(@class, "dimmed")]'))
+
+
+class Paginator(Widget):
+    """
+    Represents Paginator control that includes First/Last/Next/Prev buttons and control displaying
+    amount of items on current page and overall amount.
+
+    It is mainly used in Paginator Pane.
+    """
+    PAGINATOR_LOCATOR = './/ul[@class="pagination"]/'
+    CUR_PAGE_LOCATOR = PAGINATOR_LOCATOR + './li/span/input[@name="limitstart"]/..'
+    PAGE_BUTTON_LOCATOR = PAGINATOR_LOCATOR + './li[contains(@class, {})]/span'
+
+    def __init__(self, parent, logger=None):
+        super(Paginator, self).__init__(parent=parent, logger=logger)
```

Comment:
In this case you do not need to add a custom init. But I would rather see a root locator that will be used to specify the root element to make this widget more broadly usable. Think that the PAGINATOR_LOCATOR uses a class to locate the element, therefore there can be multiple paginators in there.